examples/train_model.py

```diff
@@ -145,6 +145,8 @@ def validate(self):
         valid_report, self.valid_world = run_eval(
             self.agent, opt, 'valid', opt['validation_max_exs'],
             valid_world=self.valid_world)
+        if opt.get('model_file'):
```

Comment:
what's this for?

Reply:
in case you want to continue training from where you stopped instead of from the best valid so far

Reply:
it won't have self.best_valid in either case will it? or will it?
i would make this an option as it doubles the space on disk

Reply:
we save the best valid result straight to the model_file arg with nothing appended to it. I'll add a flag for last valid though!